### ISSUE TITLE
vSphere - Templates are not listing from content library

### ISSUE THREAD

**Issue by ansilh** [rancher/dashboard#4302]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
- Add vSphere cloud credential
- Add a template
- Change creation method in Instance options to "Deploy from template: Content library"
- Select the content library

**Result:**

No templates were listed in the dropdown 
![image](https://user-images.githubusercontent.com/11064902/114660273-a3eb2180-9d12-11eb-9c87-544345dbfc9d.png)

**Other details that may be helpful:**

The same credentials and content library works fine in v2.4.8 (didn't check other latest v2.4.x)
In the problematic version, the below API call is not observed during the browser inspection.
```
/meta/vsphere/library-templates?cloudCredentialId=cattle-global-data:cc-b8zpf&library=my-content-libray
```

![image](https://user-images.githubusercontent.com/11064902/114660426-e7de2680-9d12-11eb-9529-3bbc24ead69b.png)

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): v2.5.7
- Installation option (single install/HA): HA

**Comment by gaktive** [comment]
Note that this was first reported in 2.5.7 along with 2.6.0.

**Comment by janeczku** [comment]
This should be backported to 2.5.x as well

**Comment by gaktive** [comment]
This should be easy to work on but UI needs an vSphere environment that has a content library to work off with debugging to isolate where to focus on.

**Comment by gaktive** [comment]
Internal ref: 3391

**Comment by richard-cox** [comment]
Fix is available using the ui `v2.6.3-patch1-rc1` release. Until there's an RC of Rancher this can be tested by
1. Bring up Rancher 2.6.3 and log in to the Dashboard
2. `Global Settings` --> `Advanced Settings`
  i.  Set `ui-index` to `https://releases.rancher.com/ui/v2.6.3-patch1-rc1/index.html`
  ii. Set `ui-offline-preferred` to `Remote`
  iii. Refresh the browser

**Comment by slickwarren** [comment]
tested on v2.6.3-patch1-rc1 using vsphere v7.0u:
* Content Library templates now are properly fetched and loaded through rancher for rke1 clusters
<img width="656" alt="Screen Shot 2022-01-21 at 10 10 44 AM" src="https://user-images.githubusercontent.com/16691014/150583030-bf34ea63-fc2c-4002-8097-8e35001876a7.png">
